docs/root/configuration/http_filters/gzip_filter.rst

```diff
@@ -0,0 +1,50 @@
+.. _config_http_filters_gzip:
+
+Gzip
+====
+Gzip is an HTTP filter which enables Envoy to compress dispatched data
+from an upstream service upon client request. Compression is useful in
+situations where large payloads need to be transmitted without
+compromising the response time.
+
+Configuration
+-------------
+* :ref:`v1 API reference <config_http_filters_gzip_v1>`
+* :ref:`v2 API reference <envoy_api_msg_filter.http.Gzip>`
+
+.. attention::
+
+  Due to a known bug in the underlying Zlib library, window bits with value
```

Comment:
The reader won't know what a "window bit" is here, maybe a half-sentence intro would help.